bin/fin

```diff
@@ -4545,6 +4546,19 @@ run_cli ()
 	fi
 }
 
+# Reload Cron Configuration
+_cron_reload ()
+{
+	echo-warning "Reloading Cron..."
+	# If crontab file is found within project add contents to user crontab file.
+	if [[ -f "$(get_project_path_dc)/.docksal/services/cli/crontab" ]]; then
+		_exec 'cat ${PROJECT_ROOT}/.docksal/services/cli/crontab | crontab -u docker -'
+		_exec 'sudo /etc/init.d/cron start 2>/dev/null'
+	else
+		echo-error "No crontab file has been found within project."
```

Comment:
Let's change to 

```bash
echo-error "No crontab file has been found in ${yellow}.docksal/services/cli/crontab${NC}."
return 1
```

to give user some clue where to put `crontab` file without even him needing to read the docs.